File: app/algorithm/AA_25-26/models/channel.py
class Channel:
    def __init__(self, channel_id, channel_name, programs):
        self.channel_id = channel_id
        self.channel_name = channel_name
        self.programs = programs

    def __repr__(self):
        return f"Channel({self.channel_id}, {self.channel_name}, Programs: {len(self.programs)})"

File: app/algorithm/AA_25-26/SchedulingAPI/Models/Channel.cs
namespace SchedulingAPI.Models
{
    public class Channel
    {
        public int ChannelId { get; set; }
        public string ChannelName { get; set; } = string.Empty;
        public List<TVProgram> Programs { get; set; } = new();

        public override string ToString()
        {
            return $"Channel({ChannelId}, {ChannelName}, Programs: {Programs.Count})";
        }
    }
}

